plugins/gojira-postgresro

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+POSTGRES_RO_PATH=$(dirname ${BASH_SOURCE[0]})/postgresro
+
+#  Add commands to usage
+function gojira-postgresro-commands {
+  cat << EOF
+[postgresro] Commands:
+  up                       gojira's up command is annotated
+
+EOF
+}
+
+function gojira-postgresro-usage {
+  cat << EOF
+
+This plugin adds a read-only replica in addition to the primary postgres database.
+
+EOF
+}
+
+replica_egg() {
+  cat << EOF
+version: '3.5'
+services:
+  # add some PG_RO_HOST and point it to the replica
+  ${GOJIRA_TARGET:-kong}:
+    environment:
+      KONG_PG_RO_HOST: postgresro
+
+  # set replica parameters on the db container
+  db:
+    environment:
+      PG_REP_USER: rep
```

Comment:
WDYT about allowing an override and making it consistent with the main db instance?

```suggestion
      PG_REP_USER: ${KONG_PG_REP_USER:-kong-replica}
```